instill/model/v1alpha/model.proto

```diff
@@ -0,0 +1,372 @@
+  // Output from a model
+  Output output = 1;
+}
+
+// TriggerModelBinaryFileUploadRequest represents a request to trigger a model
+message TriggerModelBinaryFileUploadRequest {
+  // Model name
+  string name = 1 [ (google.api.field_behavior) = REQUIRED ];
+  // Model version
+  int32 version = 2 [ (google.api.field_behavior) = REQUIRED ];
+  // Model content in byte chunk
+  bytes chunk = 3 [ (google.api.field_behavior) = REQUIRED ];
+}
+
+// TriggerModelBinaryFileUploadResponse represents a response for the output of a model
+message TriggerModelBinaryFileUploadResponse {
+  // Output from a model
+  Output output = 1;
+}
+
+// ClassificationOutput represents the output of classification task
+message ClassificationOutput {
+  // Classification cateogry
+  string category = 1;
+  // Classification score
+  float score = 2;
+}
+
+// ClassificationOutputs represents a list of classification task outputs
+message ClassificationOutputs {
+  // A list of classification outputs
+  repeated ClassificationOutput classification_outputs = 1;
+}
+
+// BoundingBox represents the bounding box data structure
+message BoundingBox {
+  // Bounding box top y-axis value
+  float top = 1;
+  // Bounding box left x-axis value
+  float left = 2;
+  // Bounding box width value
+  float width = 3;
+  // Bounding box height value
+  float height = 4;
+}
+
+// BoundingBoxObject represents a predicted bounding box object
+message BoundingBoxObject {
+  // Bounding box object category
+  string category = 1;
+  // Bounding box object scroe
+  float score = 2;
+  // Bounding box
+  BoundingBox bounding_box = 3;
+}
+
+// DetectionOutput represents the output of detection task
+message DetectionOutput {
+  // A list of buonding box objects
+  repeated BoundingBoxObject bounding_box_objects = 1;
+}
+
+// DetectionOutputs represents a list of detection task outputs
+message DetectionOutputs {
+  // A list of detection outputs
+  repeated DetectionOutput detection_output = 1;
+}
+
+// Model service responds to incoming model requests.
+service ModelService {
+
+  // Liveness method receives a LivenessRequest message and returns a
+  // LivenessResponse message.
+  // See https://github.com/grpc/grpc/blob/master/doc/health-checking.md
+  rpc Liveness(LivenessRequest) returns (LivenessResponse) {
+    option (google.api.http) = {
+      get : "/__liveness"
+      additional_bindings : [ {get : "/health/model"} ]
+    };
+  }
+
+  // Readiness method receives a ReadinessRequest message and returns a
+  // ReadinessResponse message.
+  // See https://github.com/grpc/grpc/blob/master/doc/health-checking.md
+  rpc Readiness(ReadinessRequest) returns (ReadinessResponse) {
+    option (google.api.http) = {
+      get : "/__readiness"
+    };
+  }
+
+  // CreateModel method receives a CreateModelRequest message and returns
+  // a CreateModelResponse message.
+  rpc CreateModel(CreateModelRequest) returns (CreateModelResponse) {
+    option (google.api.http) = {
```

Comment:
@pinglin there is only a unary RPC method to create model. Do we need stream RPC to upload model?